src/server/Elsa.Server.Api/Endpoints/WorkflowRegistry/ListAll.cs

```diff
@@ -34,7 +34,7 @@ public ListAll(IEnumerable<IWorkflowProvider> workflowProviders, ITenantAccessor
     }
 
     [HttpGet]
-    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(PagedList<WorkflowBlueprintSummaryModel>))]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(List<WorkflowBlueprintSummaryModel>))]
     [SwaggerOperation(
         Summary = "Returns a list of workflow blueprints.",
         Description = "Returns paginated a list of workflow blueprints. When no version options are specified, the latest version is returned.",
```

Comment:
Let's also fix the description while you're at it: ""Returns a list of workflow blueprints. When no version options are specified, the latest version is returned." 👍🏻